README.md

```diff
@@ -5,53 +5,59 @@ The USDS website is built with:
 - [Jekyll](https://jekyllrb.com/) via the [github-pages gem](https://rubygems.org/gems/github-pages)
   - See [current dependencies](https://pages.github.com/versions/) for GitHub Pages
 - [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/)
-- [Gulp](https://gulpjs.com/) / [USWDS Gulp](https://github.com/uswds/uswds-gulp)
 
 Before getting started, install the following on your system:
 
-- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/), best set up on version 2.7 for Jekyll
 - [NPM](https://github.com/npm/cli)
-- [Gulp CLI](https://gulpjs.com/docs/en/getting-started/quick-start)
+- Docker
 
-## Install and run
+## Install and run a development environment
 
 ### Initial setup
-```
-$ npm install
-$ bundle install
-$ npm start
-```
+We recommmend that you use a Docker container to run the site as a development environment because of dependency and version collisions. This allows the site to run in a contained place, minimizing setup issues.
 
-### Running and building
-#### Run locally & watch for changes
 ```
-$ npm start
-~ or ~
-$ npm run serve
+docker build -t usds-website . --no-cache
 ```
-Builds the Jekyll site, spins it up at `localhost:4000`, and watches for changes. Compiles main stylesheet with Gulp and watches for changes in `assets/stylesheets/**/*.scss`.
 
-#### Local build
+### Run locally, with and without live reload
+Run this command to have a local version on `http://localhost:4080`.
 ```
-$ npm run build
+docker run -p 4080:4000  --name usdsweb usds-website
 ```
- Builds local dev version of the Jekyll site and compiles main stylesheet with Gulp.  
 
-#### Staging build
-```
-$ npm run build-staging
+If you want to make changes and have them update on the running container
+we need to map the active source code in to the container with a volume mount and
+expose the port for livereload. This should ensure that changes you make are being
+mapped to the /app director and refresh on the running container. Note the volume mount syntax
+for `pwd` might be different on Windows or a non Bash shell. 
```

Comment:
Good to give a hint to non bash folks, but if they need the hint this probably doesn't give them what they need to solve it. Suggest something like

> In bash, `$(pwd)` inserts the path of the current working directory into the command shown. If that does not work for you, try replacing that portion of the command with the correct path.